### ISSUE TITLE
R18 and <input type="file">

### ISSUE THREAD

**Issue by dan-tripp-siteimprove** [Siteimprove/alfa#1475]
These fail R18, and I think that they should pass:
```
Fails R18: <input type="file" aria-required="true" aria-label="field 1"><br>
<br>
Fails R18: <input type="file" aria-invalid="false" aria-label="field 2"><br>
```
I think they should pass because of [this ACT rule change](https://github.com/act-rules/act-rules.github.io/issues/1573).  And [ARIA in HTML for &lt;input type="file"&gt;](https://w3c.github.io/html-aria/#el-input-file) says that the state and property allowances are "Global aria-* attributes, aria-disabled, aria-invalid and aria-required attributes."

**Comment by Jym77** [comment]
We need to look at all `<input type=...>`in ARIA in HTML and solve all of them in one go.
